pubio/components/Cards.js

```diff
@@ -23,49 +23,225 @@ export default function Card(props) {
 
   return (
     <View>
-      <TouchableOpacity
-        style={{
-          position: "absolute",
-          zIndex: 999,
-          top: 50,
-          left: 28,
-          overFlow: "hidden",
-          justifyContent: "center",
-        }}
-      >
+      <TouchableOpacity style={styles.avipos}>
         {crawlcontext[0][props.crawlIndex].subscribed.map((user, index) => {
           // display only 5 avatars
           if (index < 5) {
-            return (
-              <Avatar.Image
-                key={index}
-                style={{
-                  borderWidth: 0.7,
-                  borderRadius: 50,
-                  height: 45,
-                  width: 45,
-                  position: "absolute",
-                  zIndex: 999 - index,
-                  left: index * 20,
-                  borderColor: "lightgray",
-                  opacity: index > 0 ? 0.8 : 1,
-                  overFlow: "hidden",
-                  backgroundColor: "lightgray",
-                  shadowOffset: {
-                    width: 0,
-                    height: 4,
-                  },
-                  shadowColor: "black",
-                  shadowOpacity: 0.7,
-                  shadowRadius: 3.84,
-                  elevation: 8,
-                }}
-                source={{
-                  uri: user.profile,
-                }}
-                size={45}
-              />
-            );
+            switch (user.profile) {
```

Comment:
maybe use switch case on a source variable to dry this out?

Reply:
I was on that same line of thought, but unfortunately the require method doesn't take any variables. It has to be strictly a string 🙄

Reply:
we may be able to dry it up though, we'll have to brainstorm @tan-x